catapult/config.py

```diff
@@ -12,7 +12,7 @@
     sys.exit(1)
 
 # git config
-GIT_REPO = os.environ.get("CATAPULT_GIT_REPO", "./")
+GIT_REPO = os.environ.get("CATAPULT_TARGET_GIT_REPO") or os.environ.get("CATAPULT_GIT_REPO", "./")
```

Comment:
`CATAPULT_TARGET_GIT_REPO` and `CATAPULT_GIT_REPO` are now valid (removes the explanation from the README